pages/apis/api_differences.md.erb

```diff
@@ -0,0 +1,48 @@
+# API Differences Between REST and GraphQL
+
+Buildkite provides both a [REST API](https://buildkite.com/docs/apis/rest-api) and [GraphQL API](/docs/apis/graphql-api), but there is no full parity between the two. Certain actions can only be performed using the REST API, but not GraphQL, and vice versa.
+
+{:toc}
+
+The strengths of the GraphQL API are in complex data queries, and the strengths of the REST API are in creating and modifying records.
+
+On this page we've collected the known limitation where some API features are only available with either REST or GraphQL.
+
+## Features only available in the REST API
+
+* **ANNOTATIONS:** [retrieve and list annotations](/docs/apis/rest-api/annotations) on a build, but not create them (however, you can create the annotations using the [buildkite-agent annotate command](/docs/agent/v3/cli-pipeline) from within a job).
+* **BUILDS:** set [`branch_configuration` and `github provider configuration`](/docs/pipelines/branch-configuration#step-level-branch-filtering) on pipelines.
+* **BUILDS:** [get a build by ID](/docs/apis/rest-api/builds#get-a-build).
+* **JOBS:** [get an output of job logs](/docs/apis/rest-api/jobs#get-a-jobs-log-output).
+* **JOBS:** [retry data for jobs](/docs/apis/rest-api/jobs#retry-a-job).
+* **META:** [get a list of IPs from which Buildkite sends webhooks](/docs/apis/rest-api/meta#get-meta-information).
+* **PIPELINES:** [set provider properties](https://bk-docs-pr-1613.onrender.com/docs/apis/rest-api/pipelines#provider-settings-properties) `provider_settings` allow configuring how the pipeline is triggered based on the source code provider's events; available on pipeline for all the pipeline inputs on pipeline create.
+
+## Features only available in the GraphQL API
+
+* **AGENTS:** [get a list of agent token IDs](/docs/apis/graphql/graphql-cookbook#get-a-list-of-agent-token-ids).
+* **BUILDS:** [get all environment variables set on a build](/docs/apis/graphql/graphql-cookbook#get-all-environment-variables-set-on-a-build).
+* **BUILDS:** [increase the next build number](/docs/apis/graphql/graphql-cookbook#increase-the-next-build-number).
+* **BUILDS:** [retrieve, list, and create annotations on a build](/docs/apis/rest-api/annotations).
+* **JOBS:** [get all jobs in a given queue for a given timeframe](/docs/apis/graphql/graphql-cookbook#get-all-jobs-in-a-given-queue-for-a-given-timeframe).
+* **JOBS:** list audit log events.
+* **ORGANIZATIONS:** [remove users from an organization](/docs/apis/graphql/graphql-cookbook#delete-an-organization-member).
+* **ORGANIZATIONS:** [set up SSO](/docs/integrations/sso/sso-setup-with-graphql).
+* **ORGANIZATIONS:** manage teams - [add](/docs/apis/graphql/graphql-cookbook#add-an-existing-organization-user-to-a-team) or [remove](/docs/apis/graphql/graphql-cookbook#remove-a-team-member) a team member.
```

Comment:
Teams in general is mostly graphql only. The rest api can list teams, and can create pipelines in teams. Other team management is graphql only.